src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -145,6 +150,7 @@ function parseEndpoints(
         maxRetryDuration: "string",
       });
       triggered = { scheduleTrigger: ep.scheduleTrigger };
+      ep.labels = { ...ep.labels, "deployment-scheduled": "true" };
```

Comment:
Same as above. The label should be part of the Function, not necessarily the Endpoint (though it isn't a disaster if it is)